### ISSUE TITLE
Improve error handling when firefox isn't installed

### ISSUE THREAD

**Issue by LarsDenBakker** [puppeteer/puppeteer#6223]
When you want to use puppeteer with firefox you need to install your dependencies using an extra environment variable. If you don't do this and try to use firefox, the error message you get it is

```
Error: Could not find browser revision latest. Run "npm install" or "yarn install" to download a browser binary.
```

I think we could improve this by explaining the user that they need to set up an environent variable in order to download firefox. I'm wrapping puppeteer as part of another tool, so users will have an even harder time figuring out how to solve this. 

I can submit a PR if you think this is useful to add.

**Comment by christian-bromann** [comment]
@LarsDenBakker do you think this got resolved with #6226 ?

**Comment by LarsDenBakker** [comment]
Yes indeed. Thanks :)
